exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,14 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative"
+  }
+  else if (number >= 0) {
+    return "positive"
+  }
+}
```

Comment:
This function works in the console, but I think the convention is to finish off such function with 'else' not with 'else if'. 
Since after returning negative numbers you only have positive numbers or zero, you can simplify your function into:

function negativeOrPositive(number) {
  if (number < 0) {
    return "negative";
  }else{
    return "positive";
  }
}